device/iothub-java-client/src/main/java/com/microsoft/azure/iothub/CustomLogger.java

```diff
@@ -0,0 +1,37 @@
+package com.microsoft.azure.iothub;
+
+import org.apache.log4j.*;
+
+public class CustomLogger {
+
+    private Logger logger;
+
+    public CustomLogger(Class<?> clazz)
+    {
+        logger = Logger.getLogger(clazz);
+    }
+
+    public void LogInfo(String message, Object...params)
+    {
+        if(logger.isEnabledFor(Level.INFO))
+        {
```

Comment:
Would it be possible to add granularity for each of these messages ? specifically - ALL, debug, info, fine, severe, off, trace, warn ?